CMakeSettings.json

```diff
@@ -97,46 +86,6 @@
       "cmakeCommandArgs": "-DADD_WINDOWS_ENCLAVE_TESTS=1 -DLINUX_BIN_DIR=${env.WSL_ROOTFS}/var/tmp/build/${workspaceHash}/build/Linux-Debug/tests ",
       "buildCommandArgs": "-v",
       "ctestCommandArgs": ""
-    },
-    {
```

Comment:
The Linux-Debug and Linux-Release configurations were supposed to work with WSL. Why are these being removed? Are these broken?

Reply:
Do they actually work with WSL? If you open them up in Visual Studio, it flags lot of schema errors for those entries.

Reply:
I investigated this. Installing "Linux Workload" in Visual Studio makes Visual Studio support the scheme used by "Linux-Debug" and "Linux-Release" configurations.
These workflows support copying source over to a remove linux machine and performing builds.
So most likely they would support WSL too.